housekeeper/cli/delete.py

```diff
@@ -234,3 +236,35 @@ def file_cmd(context, yes, file_id):
         store.session.delete(file)
         store.session.commit()
         LOG.info("file deleted")
+
+
+@delete.command("sample-sheet")
+@click.option("-y", "--yes", is_flag=True, help="skip checks")
+@click.argument("flow-cell-id", type=str)
+@click.pass_context
+def delete_sample_sheet(context, yes, flow_cell_id):
+    """Delete the samplesheet for the given flow_cell."""
+    store: Store = context.obj["store"]
+    try:
+        file = store.get_files(bundle_name=flow_cell_id, tag_names=["samplesheet"]).one()
+    except MultipleResultsFound as error:
```

Comment:
Is there such a case? Or did you add this just in case? I would say (95% sure) there are no bundles with two sample sheets (after my last fix)

Reply:
Ran this in prod:
```python
hk_api: HousekeeperAPI = get_hk_api()
sample_sheet_tag: Tag = hk_api.get_tag(name="samplesheet")
sample_sheet_files = sample_sheet_tag.files
sample_sheet_bundles = [file.version.bundle.name for file in sample_sheet_files]
print(f"There are {len(set(sample_sheet_bundles))} bundles and {len(sample_sheet_files)} sample sheets")
```
```shell
There are 2149 bundles and 2149 sample sheets
```